cluster/addons/kube-proxy/kubelet-user-standalone/kube-proxy-rbac.yaml

```diff
@@ -0,0 +1,14 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: system:kubelet-user-kube-proxy-binding
+  labels:
+    addonmanager.kubernetes.io/mode: Reconcile
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: kubelet
```

Comment:
this isn't the normal kubelet user… most kubelet credentials have a client certificate in the system:nodes group

Reply:
this is the `User` we've been assuming for the kubelet in a few different places. FWIW this is the same thing we did to allow for the NPD to use the kubelet's credentials: https://github.com/kubernetes/kubernetes/pull/85014

Reply:
I have the same question mike had in https://github.com/kubernetes/kubernetes/pull/85014/files#r345580236 ... this is not the user the kubelet runs as under normal circumstances

Reply:
updated the CRB to be applied to the system:nodes group, as well